.eslintrc.js

```diff
@@ -105,5 +109,26 @@ module.exports = {
     ],
     // We don't use jasmine and this clashes with danger js
     'jest/no-jasmine-globals': 'off',
+    // New rules
+    'unicorn/no-array-reduce': 'warn',
```

Comment:
question: getting some new warnings in color-contrast-utils from this change in this very PR. Do we have a strong opinion here?

Reply:
literally none. This PR was large enough though, that I didn't want to also start literally it with a bunch of fixes to things we might actually legitimately care about.

So keeping them as a warning seemed like a reasonable compromise. We will be informed via warnings that we should in the future look at these new rules and take a stance on them at a later stage.